doc/news/_preparation_next_release.md

```diff
@@ -99,6 +99,9 @@ you up to date with the multi-language support provided by Elektra.
 ### Java
 
 - Upgraded maven dependencies for java binding _(Michael Zronek)_
+- Completely overhauled the java binding to be able to use Elektra plugins directly. For an example see the [test case](src/bindings/jna/libelektra4j/src/test/java/org/libelektra/ErrorCodeTest.java). _(Michael Zronek)_
+- The java binding now supports the [error codes](doc/decisions/error_codes.md) in a native way. All exceptions contain the necessary information. For
+  an example please take a look at the [test case](src/bindings/jna/libelektra4j/src/test/java/org/libelektra/ErrorCodeTest.java). _(Michael Zronek)_
```

Comment:
Twice the same example?

Reply:
Yes, one shows how to use the `NativePlugin` and one shows how Exceptions are used. They are both  used in the test

Reply:
It would be good to have more than one example. And examples should not try to to much. I still do not understand what the differences in the new plugin system is.

If it is better, we should migrate the old plugins (which are only small examples anyway) to the new system.

Reply:
I added another example in the README.md
I migrated the old plugins to the new Interface with added methods. Also NativeElektraPlugins implements it and works.
I removed Template&ExceptionTest as it is not really valuable